test/utils/test_zenith_angle.py

```diff
@@ -61,3 +66,122 @@ def test_zenith_angle_array():
     lon = np.array([0.0])[None, None, :]
     out = cos_zenith_angle_from_timestamp(timestamp, lon, lat)
     assert out.shape == (3, 2, 1)
+
+
+@pytest.mark.parametrize(
+    "t",
+    [
+        datetime(2020, 7, 6, 9, 0, 0),
+        datetime(2000, 1, 1, 12, 0, 0),
+        datetime(2000, 7, 1, 12, 0, 0),
+        datetime(2000, 7, 1, 12, 0, 0, tzinfo=utc),
+    ],
+)
+def test_timestamp_to_julian_centuries(t):
+    a = _datetime_to_julian_century(t)
+    b = _timestamp_to_julian_century(t.replace(tzinfo=utc).timestamp())
+    assert a == b
+
+
+def test_toa():
+    t = datetime(2000, 7, 1, 12, 0, 0, tzinfo=utc).timestamp()
+    lat, lon = 0.0, 0.0
+    ans = toa_incident_solar_radiation_accumulated(t, lat, lon)
+    assert ans >= 0
+
+
+@pytest.skip(
+    "This test requires CDS API and manual review of the tisr.png "
+    "and line .png. Consider running if modifying the tisr code."
+)
+def test_tisr_matches_cds():
+    import os
+
+    import cdsapi
+    import matplotlib.pyplot as plt
```

Comment:
matplotlib also not in `[dev]` dependencies currently.

Reply:
this is why i skipped the test.

Reply:
removed the test.